docs/SemanticARC.md

```diff
@@ -92,20 +101,28 @@ Once we are able to reason about RC Identity, the next step in implementing Sema
 
    b. If the copy_value instruction has the [take] flag associated with it, then a move is being performed and while a bit by bit copy of the value occurs, no retain_value is applied to it. The original SSA value as a result of this operation has an undefined bit value and in debugging situations could be given a malloc scribbled payload.
 
+[EE] Why would we need a copy_value [take]?
```

Comment:
The reason that I originally put in the copy_value [take] was that I was hoping that any algorithms that we implement would not need to think about updating (i.e. removing) instructions until it was complete. It would be storing state in the IR essentially. Upon reflection, I realized that it should be sufficient to just store the information in an analysis or in a side table or something like that.